files/package.json

```diff
@@ -49,14 +49,16 @@
     "@glint/environment-ember-loose": "^1.4.0",
     "@glint/environment-ember-template-imports": "^1.4.0",
     "@glint/template": "^1.4.0",
+    "@ember/app-tsconfig": "^1.0.0",
     "@ember/library-tsconfig": "^1.0.0",<% } %>
-    "@rollup/plugin-babel": "^6.0.4",
+    "@rollup/plugin-babel": "^6.0.4",<% if (typescript) { %>
+    "@types/qunit": "^2.19.12",<% } %>
     "babel-plugin-ember-template-compilation": "^2.2.5",
     "concurrently": "^9.0.1",
     "ember-qunit": "^9.0.2",
     "ember-resolver": "^13.1.0",
     "ember-source": "^6.3.0",
-    "ember-template-lint": "^7.7.0",
+    "ember-template-lint": "^7.9.0",
```

Comment:
update here needed for bugfix to enable https://github.com/ember-cli/ember-addon-blueprint/pull/55/files#diff-686c1810f8a96898e768c10b17e6557518dde610f45fda271f25244f2520f041R5 (the disabling of ceckHbsTemplateLiterals)